kod/object/active/holder/nomoveon/battler/monster/towns/barlqtwn/bqSmith.kod

```diff
@@ -28,7 +28,7 @@ resources:
       "get the impression that he is uncomfortable with strangers, but his great size and clear "
       "devotion to Kraanan makes him afraid of no one."
 
-   BQsmith_entry_welcome = "Hello %s%s, and welcome."
+   BQsmith_entry_welcome = "~kFehr'loi Qan says, \"Hello %s%s, and welcome.\""
```

Comment:
I don't understand why in general these were all changed to include the name of the NPC here, duplicating it from elsewhere.  This makes the string more complicated and would also make translation more difficult in the future.  The previous way seems a lot better.